nearcore/src/runtime/mod.rs

```diff
@@ -699,11 +701,15 @@ impl RuntimeAdapter for NightshadeRuntime {
         self.flat_state_factory.get_flat_storage_state_for_shard(shard_id)
     }
 
-    fn add_flat_storage_state_for_shard(
+    #[cfg(feature = "protocol_feature_flat_state")]
```

Comment:
where does this come from ? bad merge ?

Reply:
Good catch! I don't actually understand how this got in, will sort out once the other two PRs land